lib/commands/run.js

```diff
@@ -71,7 +71,11 @@ class RunCommand extends Command {
                 return;
             }
 
-            instance.process.error(message.error);
+            // Ghost error log happens AFTER this event
+            // REMOVE WHEN ghost is released and we bump the cli engine?
+            setTimeout(() => {
```

Comment:
See https://github.com/TryGhost/Ghost/blob/1.20.3/index.js#L44.
Logging happens to late. PR to Ghost required.
For backwards support, we should keep a timeout here?

Reply:
[Ghost PR](https://github.com/TryGhost/Ghost/pull/9440)